packages/cosmos_ui_components/lib/components/cosmos_passcode_prompt.dart

```diff
@@ -91,24 +91,27 @@ class _CosmosPasscodePromptState extends State<CosmosPasscodePrompt> {
               style: CosmosTextTheme.labelL,
             ),
             SizedBox(height: CosmosTheme.of(context).spacingXL),
-            PinPut(
-              controller: _pinPutController,
-              fieldsCount: widget.pinCount,
-              fieldsAlignment: MainAxisAlignment.center,
-              eachFieldMargin: const EdgeInsets.symmetric(horizontal: 12),
-              eachFieldConstraints: const BoxConstraints(
-                maxHeight: 10,
-                maxWidth: 10,
+            Theme(
+              data: Theme.of(context).copyWith(inputDecorationTheme: const InputDecorationTheme()),
```

Comment:
I'm not sure about this part, do we need to set up the empty `InputDecorationTheme` here?

Reply:
Correct! Since we are now moving towards CosmosTheme, this won't be needed. Nice catch!